src/test/java/org/apache/pirk/schema/query/LoadQuerySchemaTest.java

```diff
@@ -389,49 +320,42 @@ private void createDataSchema(String schemaFile) throws IOException
-      TestUtils.addElement(doc, rootElement, element1, PrimitiveTypePartitioner.STRING, "false", PrimitiveTypePartitioner.class.getName());
-
-      // element2 - -- array of Integers
-      TestUtils.addElement(doc, rootElement, element2, PrimitiveTypePartitioner.INT, "true", PrimitiveTypePartitioner.class.getName());
-
-      // element3 -- array of IP addresses
-      TestUtils.addElement(doc, rootElement, element3, PrimitiveTypePartitioner.STRING, "true", IPDataPartitioner.class.getName());
-
-      // element4 -- single byte type
-      TestUtils.addElement(doc, rootElement, element4, PrimitiveTypePartitioner.BYTE, "false", PrimitiveTypePartitioner.class.getName());
-
-      // Write to a xml file
-      TransformerFactory transformerFactory = TransformerFactory.newInstance();
-      Transformer transformer = transformerFactory.newTransformer();
-      DOMSource source = new DOMSource(doc);
-      StreamResult result = new StreamResult(file);
-      transformer.transform(source, result);
-
-      // Output for testing
-      StreamResult consoleResult = new StreamResult(System.out);
-      transformer.transform(source, consoleResult);
-      System.out.println();
-
-    } catch (Exception e)
-    {
-      e.printStackTrace();
-    }
+    DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+    DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+    Document doc = dBuilder.newDocument();
+
+    // root element
+    Element rootElement = doc.createElement("schema");
+    doc.appendChild(rootElement);
+
+    // Add the schemaName
+    Element schemaNameElement = doc.createElement("schemaName");
+    schemaNameElement.appendChild(doc.createTextNode(dataSchemaName));
+    rootElement.appendChild(schemaNameElement);
+
+    // Add the elements
+    // element1 -- single String
+    TestUtils.addElement(doc, rootElement, element1, PrimitiveTypePartitioner.STRING, "false", PrimitiveTypePartitioner.class.getName());
+
+    // element2 - -- array of Integers
+    TestUtils.addElement(doc, rootElement, element2, PrimitiveTypePartitioner.INT, "true", PrimitiveTypePartitioner.class.getName());
+
+    // element3 -- array of IP addresses
+    TestUtils.addElement(doc, rootElement, element3, PrimitiveTypePartitioner.STRING, "true", IPDataPartitioner.class.getName());
+
+    // element4 -- single byte type
+    TestUtils.addElement(doc, rootElement, element4, PrimitiveTypePartitioner.BYTE, "false", PrimitiveTypePartitioner.class.getName());
+
+    // Write to a xml file
+    TransformerFactory transformerFactory = TransformerFactory.newInstance();
+    Transformer transformer = transformerFactory.newTransformer();
+    DOMSource source = new DOMSource(doc);
+    StreamResult result = new StreamResult(file);
+    transformer.transform(source, result);
+
+    // Output for testing
+    StreamResult consoleResult = new StreamResult(System.out);
+    transformer.transform(source, consoleResult);
+    System.out.println();
```

Comment:
Yeah, would be good.  This is not changed from the original file, so I'm not going there right now as it's not immediately obvious to me how to transform the XML model onto the console stream.